packages/create-svelte/bin.js

```diff
@@ -1,150 +1 @@
-#!/usr/bin/env node
```

Comment:
This should still be kept, or at least windows will prompt to open the script in a text editor.

Reply:
thanks. done